shared/profile/wot-author/index.tsx

```diff
@@ -172,24 +299,44 @@ const VerificationChoice = (props: {
   return (
     <Kb.Box2 direction="horizontal" alignSelf="stretch" alignItems="center">
       <Kb.Box2
+        key="colorbar"
         direction="vertical"
         alignSelf="stretch"
         style={{backgroundColor: color, flexShrink: 0, width: 6}}
       />
-      <Kb.RadioButton
-        label={
-          <Kb.Text type="Body" style={Styles.globalStyles.flexOne}>
-            {text}
-          </Kb.Text>
-        }
-        selected={props.selected}
-        onSelect={props.onSelect}
-        style={styles.choiceRadio}
-      />
+      <Kb.Box2 direction="vertical" alignSelf="stretch" style={Styles.globalStyles.flexOne}>
+        <Kb.Box2 direction="horizontal" alignSelf="stretch" alignItems="center">
+          <Kb.RadioButton
+            label={
+              <Kb.Text type="Body" style={Styles.globalStyles.flexOne}>
+                {text}
+              </Kb.Text>
+            }
+            selected={props.selected}
+            onSelect={props.onSelect}
+            style={styles.choiceRadio}
+          />
+        </Kb.Box2>
+        {props.children}
+      </Kb.Box2>
     </Kb.Box2>
   )
 }
 
+// Store checkedness for a list of checkboxes.
+function useCheckboxesState<T>(items: T[]): (T & {checked: boolean; onCheck: (_: boolean) => void})[] {
+  const initial = items.map(() => false)
```

Comment:
you're creating this list on every render. Not a big deal if `items` isn't too long, but maybe move it into line 330's ternary operator